bin/run_web.sh

```diff
@@ -24,11 +24,10 @@ set -e
 
 # Launch the web-app
 gunicorn \
-    --preload \
```

Comment:
We can't use preload by default anymore because it messes up the order of monkey patching. Fun.